src/platform/utilities/feature-toggles/featureFlagNames.json

```
@@ -256,5 +256,7 @@
   "virtualAgentEnableMsftPvaTesting": "virtual_agent_enable_msft_pva_testing",
   "virtualAgentEnableNluPvaTesting": "virtual_agent_enable_nlu_pva_testing",
   "yellowRibbonAutomatedDateOnSchoolSearch": "yellow_ribbon_automated_date_on_school_search",
-  "toggleVyeAdressDirectDepositForms": "toggle_vye_adress_direct_deposit_forms"
+  "toggleVyeAdressDirectDepositForms": "toggle_vye_adress_direct_deposit_forms",
+  "toggleVyeAddressDirectDepositFormsInProfile": "toggle_vye_adress_direct_deposit_forms_in_profile",
```

Looks like you fixed the left side key spelling, but the value string still has the typo.

```suggestion
  "toggleVyeAddressDirectDepositFormsInProfile": "toggle_vye_address_direct_deposit_forms_in_profile",
```

---

Oops need to fix that in the vets-api repo first 😖